src/dosbox.cpp

```diff
@@ -444,6 +444,19 @@ void DOSBOX_Init(void) {
 	secprop->AddInitFunction(&TIMER_Init);//done
 	secprop->AddInitFunction(&CMOS_Init);//done
 
+	const char *verbosity_choices[] = {"high",   "medium", "low",
+	                                   "silent", "auto",   0};
+	Pstring = secprop->Add_string("startup_verbosity",
+	                              Property::Changeable::OnlyAtStart, "high");
+	Pstring->Set_values(verbosity_choices);
+	Pstring->Set_help("Controls verbosity prior to displaying the program:\n"
+	"       | Show Splash | Show Welcome | Show Early Stdout |\n"
+	"high   |     yes     |     yes      |       yes         |\n"
+	"medium |     no      |     yes      |       yes         |\n"
+	"low    |     no      |     no       |       yes         |\n"
+	"silent |     no      |     no       |       no          |\n"
```

Comment:
I like quiet even more; suggest we just go with that and drop silent (I would hazard that `quiet` is the most commonly used lowest verbosity level in modern software).

Reply:
Proposal for a design change here too: what do you think if `quiet` was made stronger, and guaranteed no stdout? 

Right now, I'm trying to be cautious and only quieting up to the first executable- which, like you noticed, isn't fool proof in some cases. 

I think users of `quiet` might prefer full suppression of text output before their game launches. ( If they have any issues, then they will simply Ctrl-F9 and diagnose it with normal verbosity, anyway)

This is very important in retropie, because if any text shows in dosbox then you briefly see the underlying tty frame buffer between mode changes, which is /very/ ugly.

Reply:
Yeah I wanted to suggest only `quiet` initially, but then I looked at some tools I know provide `--quiet` option - e.g. GNU make has: 
```
-s, --silent, --quiet
            Silent operation; do not print the commands as they are executed.
```
About suppressing more text: I think we should not overdo it and leave it for a future PR; some games start with text menus or with other text-based mechanisms - if users will set quiet option in their global .conf file, it might break experience… 

I had an idea about *not creating window at all* for as long as text output is being emulated - and then create it only when the first graphical mode is triggered, but it would be even more invasive change. let's stick to a simple solution for now and then we'll experiment a bit and gradually improve it :)

Personally, I will probably use `quiet` already instead of manually marking all autoexec lines with `@` in my .conf files…

Reply:
> About suppressing more text: I think we should not overdo it and leave it for a future PR; some games start with text menus or with other text-based mechanisms - if users will set quiet option in their global .conf file, it might break experience…

Roger that!

> I had an idea about not creating window at all for as long as text output is being emulated - and then create it only when the first graphical mode is triggered, but it would be even more invasive change. let's stick to a simple solution for now and then we'll experiment a bit and gradually improve it :)

 I really like this approach, especially if you have some thoughts of how to do it :-)  

Ok - I think I'm all set.  Will get these changes in tonight or tomorrow AM. 
Thanks for helping guide this feature @dreamer; I think it's going to be appreciated by fellow power-users out there.